src/components/CreateEncounter/GeneralInformation.js

```diff
@@ -25,7 +25,7 @@ const Value = styled.div`
   font-weight: 700;
 `;
 
-const GeneralInformation = ({ data, dispatch }) => {
+const GeneralInformation = ({ data, dispatch, hidePrescription }) => {
   const onCall = useCallback(
```

Comment:
nit: you can rename prop hidePrescription with showPrescription and use in code without !